src/status_im/contexts/profile/settings/screens/notifications/view.cljs

```diff
@@ -66,76 +70,125 @@
         on-change (rn/use-callback
                    #(toggle-messenger-notifications messenger-notifications-enabled?)
                    [messenger-notifications-enabled?])]
-    {:blur?        true
-     :title        (i18n/label :t/allow-messenger-notifications)
-     :action       :selector
-     :action-props {:on-change (when-not disabled? on-change)
-                    :disabled? disabled?
-                    :checked?  messenger-notifications-enabled?}}))
+    (cond-> {:blur?        true
+             :title        (i18n/label :t/allow-messenger-notifications)
+             :action       :selector
+             :action-props {:on-change (when-not disabled? on-change)
+                            :disabled? disabled?
+                            :checked?  messenger-notifications-enabled?}}
+      platform/android?
+      (assoc :title             (i18n/label :t/allow-messages-and-communities-notifications)
+             :description       :text
+             :description-props {:text (i18n/label
+                                        :t/allow-messages-and-communities-notifications-description)}))))
 
 (defn news-notifications-setting
   [{:keys [notifications-enabled? news-notifications-enabled?]}]
   (let [disabled? (not notifications-enabled?)
         on-change (rn/use-callback
                    #(toggle-news-notifications news-notifications-enabled?)
                    [news-notifications-enabled?])]
-    {:blur?        true
-     :title        (i18n/label :t/allow-news-notifications)
-     :action       :selector
-     :action-props {:on-change (when-not disabled? on-change)
-                    :disabled? disabled?
-                    :checked?  news-notifications-enabled?}}))
+    (cond-> {:blur?        true
+             :title        (i18n/label :t/allow-news-notifications)
+             :action       :selector
+             :action-props {:on-change (when-not disabled? on-change)
+                            :disabled? disabled?
+                            :checked?  news-notifications-enabled?}}
+      platform/android?
+      (assoc :description       :text
+             :description-props {:text (i18n/label :t/allow-news-notifications-description)}))))
+
+(defn- settings-group-item
+  [item & _rest]
+  [quo/category
+   {:blur?           true
+    :list-type       :settings
+    :container-style styles/settings-group-item-container
+    :data            [item]}])
+
+(defn- messenger-notifications-settings
+  [notifications-settings]
+  (let [{:keys [action-props title]} (messenger-notifications-setting notifications-settings)]
+    [rn/pressable
+     {:on-press (:on-change action-props)
+      :style    styles/settings-group-container}
+     [rn/view
+      {:style styles/settings-group-header}
+      [quo/text {:style {:flex 1}} title]
+      [quo/selectors
+       {:type      :toggle
+        :checked?  (:checked? action-props)
+        :on-change (:on-change action-props)}]]
+     [rn/view {}
```

Comment:
could you pls elaborate on why empty view is needed

Reply:
ah good catch 🙌 
I tried removing it, and it doesn't seem to be needed, ty!